test/model/store_test.dart

```diff
@@ -1191,6 +1194,20 @@ void main() {
         checkLastRequestFcm(token: '456def');
       }
     }));
+
+    testAndroidIos('use default appId if packageInfo is missing', () => awaitFakeAsync((async) async {
```

Comment:
This test won't be needed after removing the fallback.

So, instead let's have a test that uses sets different app ID at start (using `testBinding.packageInfoResult = PackageInfo(…)`) and ensure that that app ID is observed in the request.